python/src/equistore/operations/allclose.py

```diff
@@ -63,6 +149,71 @@ def allclose_raise(
     on where the two :py:class:`TensorMap` differ. See :py:func:`allclose` for
     more information on which :py:class:`TensorMap` are considered equal.
 
+    Here are examples using this function:
+
+    >>> from equistore import Labels
+    >>> # Create simple block, with one np.nan value
+    >>> block1 = TensorBlock(
+    ...     values=np.array([
+    ...          [1, 2, 4],
+    ...          [3, 5, np.nan],
+    ...      ]),
+    ...     samples=Labels(
+    ...          ["structure", "center"],
+    ...          np.array([
+    ...              [0, 0],
+    ...              [0, 1],
+    ...          ]),
+    ...      ),
+    ...     components=[],
+    ...     properties=Labels(
+    ...         ["properties"], np.array([[0], [1], [2]])
+    ...      ),
+    ... )
+    ...
+    >>> # Create a second block that differs from block1 by 1e-5 in its
+    >>> # first value
+    >>> block2 = TensorBlock(
+    ...     values=np.array([
+    ...          [1+1e-5, 2, 4],
+    ...          [3, 5, np.nan],
+    ...      ]),
+    ...     samples=Labels(
+    ...          ["structure", "center"],
+    ...          np.array([
+    ...              [0, 0],
+    ...              [0, 1],
+    ...          ]),
+    ...      ),
+    ...     components=[],
+    ...     properties=Labels(
+    ...         ["properties"], np.array([[0], [1], [2]])
+    ...      ),
+    ... )
+    ...
+    >>> #Create tensors from blocks, using same keys
+    >>> keys = Labels(names=["key"], values=np.array([[0]]))
+    ...
+    >>> tensor1 = TensorMap(keys, [block1])
+    >>> tensor2 = TensorMap(keys, [block2])
+    ...
+    >>> # Call allclose_raise, which should return a ValueError because:
+    >>> # 1. The two NaNs are not considered equal of the two tensors is 1e-5,
+    >>> # 2. The difference between the first value in the blocks
+    >>> #    which is greater than the default rtol of 1e-13
```

Comment:
Should it be like this? "The difference [...] is greater than [...]"
```suggestion
    >>> # is greater than the default rtol of 1e-13
```

Reply:
Yes, will also commit this.